src/app/components/module-overview-component/module-overview.component.ts

```diff
@@ -47,7 +47,7 @@ export class ModuleOverviewComponent implements AfterContentInit, OnDestroy {
 
         if (this.selectedCurriculumId > 0) {
           curricula.forEach(cur => {
-            if (cur.id === this.selectedCurriculumId) {
+            if (cur.id == this.selectedCurriculumId) {
```

Comment:
Why?

Reply:
Because cur.id is a string "1" and this.selectedCurriculumId is an integer 1.
Three equals would fail